sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidSqlCalcitePlannerModule.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.sql.calcite.planner;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import com.google.inject.Scopes;
+import org.apache.calcite.schema.SchemaPlus;
+import org.apache.druid.guice.JsonConfigProvider;
+
+/**
+ * The module responsible for provide bindings for the Calcite Planner.
+ */
+public class DruidSqlCalcitePlannerModule implements Module
```

Comment:
IMO CalcitePlannerModule is fine / better. We don't use Calcite for anything other than Druid SQL so it's not ambiguous.

Reply:
👍